src/static/js/change_user_settings.js

```diff
@@ -79,6 +87,7 @@ $(function() {
                         options: '<option value="" data-style="background-image: url(static/images/flags/' + config.str("Locale") + '.png)">' + _("(use system)") + '</option>' + 
                             this.two_pair_options(controller.locales, true) },
                     { post_field: "quicklinksid", label: _("Quicklinks"), type: "selectmulti", options: change_user_settings.quicklink_options() },
+                    { post_field: "favouritereportsid", label: _("Favourite Reports"), type: "selectmulti", options: { displayfield: "TITLE", rows: controller.reports} },
```

Comment:
again, really minor thing, prefer the name quickreportsid and Quick Reports!

Reply:
Already done.